content/docs/1.7.0/snapshots-and-backups/csi-volume-clone.md

```diff
@@ -41,6 +43,38 @@ spec:
 > In addition to the requirements listed at [CSI volume cloning](https://kubernetes.io/docs/concepts/storage/volume-pvc-datasource/),
 > the `cloned-pvc` must have the same `resources.requests.storage` as the `source-pvc`.
 
+
+## Clone a Volume Using the Longhorn UI
+
+### Clone a volume
+1. Go to the **Volume** screen.
+2. Select a volume, and then click **Clone Volume** in the **Operation** menu.
+3. (Optional) Configure the settings of the new volume.
+4. Click **OK**.
+
+### Clone a Volume Using a Snapshot
+1. Go to the **Volume** screen.
```

Comment:
```suggestion
1. Go to the **Volume** page.
```

Reply:
In many doc style guides, "page" is used for websites while "screen" is used for program consoles. I use screen when writing about the Longhorn and Harvester UIs.

Microsoft Style Guide:
- [page](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/page)
- [screen](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/s/screen)